benchmark/benchmark.py

```diff
@@ -251,8 +251,8 @@ def rprint(*args, **kwargs):
         if action in ["cleanup", "generate", "all"]:
             cleanup()
 
-        train_examples = 10000000
-        test_examples = 10000000
+        train_examples = 1_000_000
```

Comment:
Could also be just `10 ** 6`

Reply:
Thanks, I agree, but I think it's clearer this way...